Cars/Repositories/Employee/FakeEmployeeRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cars.Model;
+using Cars.Service;
+
+namespace Cars.Repositories
+{
+    public class FakeEmployeeRepository : IEmployeeRepository
```

Comment:
We named our repositories according to the pattern "Fake<model_name>Dao" and the interface "I<model_name>Dao"